plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx

```diff
@@ -137,15 +138,19 @@ export const SettingsPaymentsMain = () => {
 			const paymentSettings = select( paymentSettingsStore );
 
 			return {
-				providers: paymentSettings.getPaymentProviders( storeCountry ),
+				providers:
+					paymentSettings.getPaymentProviders( businessCountry ),
```

Comment:
All state solvers receive the business country since they all defer to resolving the entire providers list, and that is business location dependent.